wiki/Kiai_time/id.md

```diff
@@ -0,0 +1,21 @@
+stub: true
+tags:
+   - timing section
+   - kiai mode
+   - waktu
+---
+
+# Waktu Kiai
+*Untuk peraturan seputar waktu kiai, lihat: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+**Kiai Time** (atau *kiai*) adalah istilah yang diberikan untuk bagian waktu "khusus" untuk membantu menekankan bagian dari [beatmap](/wiki/Beatmap). Pemain dapat mengetahui apakah bagian dari beatmap memiliki bagian waktu kiai jika mereka melihat efek visual tambahan di latar belakang.
+
+Saat kiai aktif, semua [objek](/wiki/Hit_object) yang dipukul akan berkedip mengikuti [irama ketukan utama](/wiki/Beatmap_Editor/Timing) dan bintang akan meledak dari kursor saat [hit circle](/wiki/Hit_object/Hit_circle) dipukul. Jika tidak, mereka akan jatuh dari kursor saat [slider](/wiki/Hit_object/Slider) atau [spinner](/Wiki/Hit_object/Spinner) diikuti.
```

Comment:
```suggestion
Saat kiai sedang aktif, semua [hit object](/wiki/Hit_object) yang dikenai akan berkedip mengikuti [main beat](/wiki/Beatmap_Editor/Timing) dan bintang akan meledak dari kursor saat [hit circle](/wiki/Hit_object/Hit_circle) dikenai. Jika tidak, mereka akan jatuh dari kursor saat kursor berada pada [slider](/wiki/Hit_object/Slider) atau [spinner](/Wiki/Hit_object/Spinner).
```

Reply:
You don't need to translate the "hit object" part, would be nice if you leave it intact ^^

I believe the "main beat" part shouldn't be translated since it's a music term and can get confusing when being translated

Reply:
ok